### DISCUSSION TITLE
How do we change the object that a hair mod is supposed to change?

### DISCUSSION THREAD

**Discussion by orenji** [xivdev/Penumbra#263] [General]
For example, if a hair mod requires Hair 3 on Viera, how can I change it so it links to Hair 9 instead?

If possible, I'm hoping for a guide on how to do this please or a brief explanation on what exactly I should be editing so it works. I tried changing the json file so it points to Hair 9 as the source in File Swaps but the character looks bald when I select Hair 9 in-game. Any help would be appreciated :)

I found a similar question being asked by someone else here but they only got a response about whether it was possible or not and it was very vague.

**Comment by NovaNami** [comment]
There's a guide on how to do it, but it's not exactly the easiest thing to do. You basically have to edit all the paths manually. https://reniguide.info/#itemconversion

**Reply by NovaNami** [reply]
If you need help with it, me and a few other people usually hang out in the Penumbra discord (https://discord.gg/kVva7DHV4r ) and help people with stuff like this

**Reply by orenji** [reply]
Apologies for the late response! Thank you so much for the response and offer to help <3
